api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java

```diff
@@ -753,7 +753,7 @@ public void reviewIdVerification_sendsEmail() throws Exception {
 
   @Test
   public void testUpdateNihToken() {
-    doNothing().when(fireCloudService).postNihCallback(any());
```

Comment:
It's a little disconcerting that we don't have any tests explicitly checking the logic of the era commons sync.

What I'd hope to see would be something akin to the testSyncTraining method below – which explicitly walks through a sync call, with a mocked-out badge response, and verifies that we update the user data model accordingly.

Feel free to push back if there's some reason it's harder to test the NIH service in the same way. But for consistency, it would be nice to have something in the same pattern.

Reply:
Will add that.